src/style.css

```
@@ -1,3 +1,64 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 10px;
+  font-weight: 500;
+  line-height: 14px;
+  text-transform: uppercase;
+  font-style: normal;
+
+  --main-color: #00acdc;
```

```suggestion
  --color--blue: #00acdc;
```
This color doesn't looks like main )